src/engine/enginebuffer.h

```diff
@@ -91,9 +91,17 @@ class EngineBuffer : public EngineObject {
   public:
     enum SeekRequest {
         SEEK_NONE = 0x00,
-        SEEK_PHASE = 0x01,
-        SEEK_EXACT = 0x02,
-        SEEK_STANDARD = 0x03, // = (SEEK_EXACT | SEEK_PHASE)
+        SEEK_PHASE = 0x01, // This is set to force an in-phase seek.
+        SEEK_EXACT = 0x02, // This is used to seek to position regardless of
+                           // if Quantize is enabled.
+        SEEK_EXACT_PHASE = 0x03, // This is an artificial state that happens if
```

Comment:
SEEK_EXACT_PHASE = SEEK_EXACT | SEEK_PHASE,
SEEK_STANDARD_PHASE = SEEK_STANDARD | SEEK_PHASE,
would be more explicit for the derived literals.